### PR TITLE
[Choreo-Connect] Support header policies for response flow. Update method rewrite.

### PR DESCRIPTION
**Pull request by suksw**
## Purpose
Update the following in Policies for Choreo Connect 
-  Support policies addHeader, removeHeader for response flow. 
-  Update method rewrite policy

## Related PRs
https://github.com/wso2/apim-apps/pull/295
https://github.com/wso2/product-microgateway/pull/2983

## Related Issues
https://github.com/wso2/product-microgateway/issues/2756

### CODE REVIEW

**Review by renuka-fernando — DISMISSED** [review]

File: modules/distribution/resources/operation_policies/specifications/ccChangeHTTPMethod_v1.json [hunk 1]
```diff
@@ -5,15 +5,6 @@
...
-    {
-      "name": "currentMethod",
-      "displayName": "Current HTTP Method",
-      "description": "Method of the HTTP request to be updated",
-      "validationRegex": "^(GET|PUT|POST|PATCH|DELETE|HEAD|OPTIONS)$",
-      "type": "Enum",
-      "allowedValues" : ["GET","PUT","POST","PATCH","DELETE","HEAD","OPTIONS"],
-      "required": true
-    },
     {
```
Comment (pubudu538) [inline modules/distribution/resources/operation_policies/specifications/ccChangeHTTPMethod_v1.json:8]: I guess we have versions in the API policy as well. I am not sure whether we can simply change this or we should create v2 of the same policy.

@1akshitha

Comment (1akshitha) [reply modules/distribution/resources/operation_policies/specifications/ccChangeHTTPMethod_v1.json:8]: Policy version support is still not visible in the UI and is not included in the 4.2.0 release.

Comment (1akshitha) [reply modules/distribution/resources/operation_policies/specifications/ccChangeHTTPMethod_v1.json:8]: we have added the version support. @CrowleyRajapakse have we added the version support to master branch? If so, we can add it as a v2.

**Comment by suksw** [comment]
This PR is currently onHold. Keeping this PR on hold until the policy version changes mentioned by @1akshitha are made to both carbon-apimgt and apim-apps.

**Review by 1akshitha — APPROVED** [review]

**Pull request merged**